### PR TITLE
Add animation-composition

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

File: feature-group-definitions/animation-composition.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: animation-composition
```
Comment (ddbeck) [inline feature-group-definitions/animation-composition.yml:1]:
```suggestion
name: animation-composition
description: The `animation-composition` CSS property chooses which composite operation to use when more than one animation applies to a property.
```

**Review by ddbeck — APPROVED** [review]
Approved, plus my bonus attempt at trying to describe this

File: feature-group-definitions/animation-composition.yml (hunk 1 above)
Comment (foolip) [reply feature-group-definitions/animation-composition.yml:1]: Should that end with "applies to an element"?

Comment (ddbeck) [reply feature-group-definitions/animation-composition.yml:1]: Implicitly it's an element too, but if I understand correctly, `animation-composition` only comes into play when two animations apply to an element that affect the same property. Or to turn that around, if you apply two animations to an element but one changes `background-color` and the other changes `transform`, then `animation-composition` doesn't matter.

Though now that I've written this, we could tighten it up a bit:

```suggestion
name: animation-composition
description: The `animation-composition` CSS property chooses how to combine animations that affect the same property.
```

Comment (foolip) [reply feature-group-definitions/animation-composition.yml:1]: Ah, yes, the thing being animated is a property, which applies to an element. Let's go with your shorter description.

**Pull request merged**